user-guide/Basic_Functionality/Elements/Working_with_elements/Adding_Elements/HTTPS_Connection.md

```diff
@@ -13,3 +13,33 @@ For HTTP(S) connections, you can specify the following connection settings while
 - **Bus address**: The bus address of the device. This is not always required. If the proxy server has to be bypassed, specify *bypassproxy*.
 
 - **Network**: The network interface (NIC). If only one network interface is available on the DMA, it is automatically selected.
+
+- **Insecure HTTPS**: From DataMiner 10.4.12/10.5.0 onwards<!--RN 40877-->, SSL/TLS certificates are validated by default for all newly created HTTP elements.
+
+  - To disable certificate validation for an element created after a 10.4.12/10.5.0 upgrade, enable the *Insecure HTTPS* option.
+
+  - To enable certificate validation for an HTTP element created before a 10.4.12/10.5.0 upgrade, disable the *Insecure HTTPS* option.
+
+  When the *Insecure HTTPS* setting is enabled, in case an HTTP connector polls an HTTPS endpoint:
+
+  - DataMiner will ignore invalid certificates in the following cases:
+
+    - The server certificate is expired.
+
+    - The server certificate is revoked.
+
+    - The common name of the server certificate does not match the server name to which DataMiner is sending the request.
+
+    - The certificate is issued by a Certificate Authority that is not trusted by the DataMiner Agent.
+
+  - DataMiner will block communication in the following cases:
+
+    - The server is offering a non-server certificate.
+
+    - The server certificate is signed by a weak signature.
+
+  > [!NOTE]
+  >
+  > - To skip SSL/TLS certificate validation for all elements that share the same *protocol.xml* file, set the `InsecureHttps` element to true in the `PortSettings` element of the *protocol.xml* file.
+  > - To skip SSL/TLS certificate validation when using multi-threaded HTTP communication, set `requestSettings[6]` to true when building the HTTP request in a QAction. For more information, see [Setting up multi-threaded HTTP communication in a QAction](xref:AdvancedMultiThreadedTimersHttp).
```

Comment:
@PedroDebevere, it seems that the `InsecureHttps` element might need to be added to [this page](https://docs.dataminer.services/develop/schemadoc/Protocol/Protocol.PortSettings.html) in the development guide, and it may also benefit from its own subpage. Additionally, [this page about setting up multi-threaded HTTP communication](https://docs.dataminer.services/develop/devguide/Connector/AdvancedMultithreadedTimersHttp.html) might need an update to include `requestSettings[6]`. 

Could you create a pull request for this? Thanks in advance!

Reply:
@EdithLansens Since the feature has been merged, it has been decided that this will be renamed to "SkipCertificateVerification" and it is a blocking change for the release.
This change is currently in code review, the associated release note is 41285.

Reply:
PR created: #3916